test/rest-coercion/jsonform-geopoint.suite.js

```diff
@@ -0,0 +1,86 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-remoting
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var GeoPoint = require('loopback-datatype-geopoint');
+var jsonFormContext = require('./_jsonform.context');
+
+module.exports = function(ctx) {
+  ctx = jsonFormContext(ctx);
+  var EMPTY_BODY = ctx.EMPTY_BODY;
+  var ERROR_BAD_REQUEST = ctx.ERROR_BAD_REQUEST;
+  var verifyTestCases = ctx.verifyTestCases;
+
+  describe('json form - geopoint - required', function() {
+    // See verifyTestCases' jsdoc for details about the format of test cases.
+    verifyTestCases({ arg: 'arg', type: 'geopoint', required: true }, [
+      // Valid values
+      [{ arg: { lat: 2.5, lng: 3.2 }}, new GeoPoint(2.5, 3.2)],
+      [{ arg: [2.5, 3.2] }, new GeoPoint(2.5, 3.2)],
+
+      // Empty values trigger ERROR_BAD_REQUEST
+      [EMPTY_BODY, ERROR_BAD_REQUEST],
+      [{ arg: null }, ERROR_BAD_REQUEST],
+      [{ arg: '' }, ERROR_BAD_REQUEST],
+      [{ arg: {}}, ERROR_BAD_REQUEST],
+      [{ arg: [] }, ERROR_BAD_REQUEST],
+    ]);
+  });
+
+  describe('json form - geopoint - optional', function() {
+    // See verifyTestCases' jsdoc for details about the format of test cases.
+    verifyTestCases({ arg: 'arg', type: 'geopoint' }, [
+      // Empty values
+      [EMPTY_BODY, undefined],
+      // [{ arg: null }, null],
+      // [{ arg: {}}, {}],
+      // [{ arg: [] }, {}],
```

Comment:
Why are these three lines commented out? Either un-comment and keep them, or remove them completely.